### PR TITLE
Bug 1229635 - use make test-integration command for integration test, r=Ricky

### PR DESCRIPTION
**Pull request by gasolin**
…

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=65ba1a62ac4a6d5f47ebae64808bd9fd76aa886b

File: README.md [hunk 1]
```diff
@@ -86,35 +86,39 @@ tests run on nodejs and you'll need nodejs>=v0.12 and npm>=v2.0 installed.
...
 
-#### Invoking tests for a specific app
+#### Invoking specific test files
 
 ```sh
-APP=<APP> npm run marionette
+TEST_FILES="/abs/path/to/some_test.js /abs/path/to/other_test.js" make test-integration
 ```
 
-#### Skipping a test file
```
Comment (gasolin) [inline README.md]: move `Skipping a test file` section to MDN

**Comment by zutiltaken** [comment]
I can't download the file gaia. .

File: README.md [hunk 2]
```diff
@@ -86,35 +86,39 @@ tests run on nodejs and you'll need nodejs>=v0.12 and npm>=v2.0 installed.
...
-SKIP_TEST_FILES="/abs/path/to/skipped_test.js /abs/path/to/other/skipped_test.js" npm run marionette
+BUILDAPP=device make test-integration
 ```
 
 #### Moar things
 
 + `VERBOSE=1` pipes gecko logs to your command line process for debugging.
 
-+ If you don't have a b2g/ folder one will be downloaded for you.
++ If you don't have a b2g/ folder, one will be downloaded for you.
```
Comment (gaye) [inline README.md]: This should now say `firefox/` @gasolin since we're using mulet, right?

Comment (gasolin) [inline README.md]: good catch, will update that

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9d10f191cf4e6130bfe20515796f452020b0e017

File: README.md [hunk 3]
```diff
@@ -86,37 +86,41 @@ tests run on nodejs and you'll need nodejs>=v0.12 and npm>=v2.0 installed.
...
-#### Skipping a test file
+##### Running tests on device
 
+You can run tests on device by plugging in your phone and adding the BUILDAPP=device to the make command:
 ```sh
-SKIP_TEST_FILES="/abs/path/to/skipped_test.js /abs/path/to/other/skipped_test.js" npm run marionette
+BUILDAPP=device make test-integration
 ```
 
 #### Moar things
```
Comment (rickychien) [inline README.md]: Not sure what is Moar means, is it a typo error?

File: README.md [hunk 4]
```diff
@@ -86,37 +86,41 @@ tests run on nodejs and you'll need nodejs>=v0.12 and npm>=v2.0 installed.
...
 
 + `VERBOSE=1` pipes gecko logs to your command line process for debugging.
 
-+ If you don't have a b2g/ folder one will be downloaded for you.
++ If you don't have a b2g_sdk/ folder, one will be downloaded for you.
   This can be problematic if you're offline. You can symlink a
-  b2g-desktop directory to b2g/ in gaia to avoid the download.
+  b2g-desktop directory to b2g_sdk/ in gaia to avoid the download.
```
Comment (rickychien) [inline README.md]: IIRC, user is able to specify various run-time to execute marionette testing. Default is set to mulet which is downloaded in gaia/firefox folder and b2g is set to gaia/b2g (I haven't checked it please correct me if I am wrong).

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=80aa976f0de1abecff5fe19f483b8b024aacfd09

File: README.md (hunk 4 above)
Comment (gasolin) [inline README.md]: let's move detail in MDN document instead of in README, to not overload new commers

**Pull request merged**